keyboards/lyra/lyra.c

```diff
@@ -0,0 +1,305 @@
+ /* Copyright 2021 Domanic Calleja
```

Comment:
This entire file should look like this: 
https://gist.github.com/drashna/77e332533bef4d539ec9a9a3b90a6cb0

Reply:
have removed the duplications